parser/shared/src/test/scala/hmda/parser/util/FITestData.scala

```diff
@@ -4,6 +4,8 @@ object FITestData {
 
   val tsDAT =
     "100110132019201501171330 201320-1177984  551  SMALL BANK USA, NA            1850 TYSONS BLVD., SUITE 50             MCLEAN                   VA22102     BIGS USA, INC.                412 THIRD AVENUE                        NEW YORK                 NY10012     Bob Smith                     555-555-5555999-999-9999bob.smith@bank.com                                                "
+  val tsDAT_noParent =
```

Comment:
Agreed.  I think `tsDATNoParent` is better...or `tsDatNoParent`, but then you have to change the other variable name to make it consistent. :)  Either is fine by me.